bundler.d/sidekiq.rb

```diff
@@ -0,0 +1,4 @@
+group :sidekiq do
```

Comment:
If this a dependency of dynflow, would the group not be better off being dynflow?

Reply:
I guess you might say you may want to have Foreman without sidekiq and that is fine and in that case `bundle install --without sidekiq` makes sense. If the group's name dynflow, running `bundle install --without dynflow` would give you foreman with dynflow but without sidekiq, which would be confusing.

Reply:
I believe we want to switch `ActiveJob`s in the future to use sidekiq as well, therefore sidekiq group, ideally it would result in dynflow use the threaded executor.

Reply:
Sorry I got lost there. With this change I can now have:

 a) Foreman
 b) Foreman with dynflow
 c) Foreman with sidekiq
 d) Foreman with sidekiq and dynflow

Is that correct?

Reply:
Now we still don't support c) and d) (`ActiveJob` directly through sidekiq, we run them through the dynflow now) but we want to run them directly on sidekiq in the future, if the sidekiq is enabled.
You can have:
a) dynflow with sidekiq - as we have now, or
b) dynflow without sidekiq

In the future:
c) Foreman's native jobs (rails native jobs) on sidekiq

Reply:
Can you have Foreman with dynflow and dynflow-sidekiq running at the same time?

Reply:
Yes, functionally it would be equivalent to setting `EXECUTORS_COUNT` to 2 in the old deployment. That would mean two executors with tasks load balanced between them.

Reply:
I think where I am getting lost is in whether dynflow-sidekiq is the new default or not. If it is, then all this ought just live in the dynflow group IMO. I feel like I have heard that it will be off by default and on by default. Given there are side and IRC conversations happening, stating clearly what the 1.24 plan is would be appreciated.

Reply:
I mean the sidekiq group give us what we want. Now it states you can choose if to use sidekiq or not. And I believe that once you've got sidekiq in, you'll want to use it for `ActiveJob`s

Reply:
As for the default now the plan is default on, but that could change as we're really close to branching.

Reply:
renamed.